<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #e6f0fa;</string>
   </property>
   <widget class="QLabel" name="originalLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4a90e2; background-color: #f7fbff;</string>
    </property>
    <property name="text">
     <string>Original Image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filteredLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #2a6bbf; background-color: #f7fbff;</string>
    </property>
    <property name="text">
     <string>Filtered Image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4a90e2; color: white; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>540</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #357ABD; color: white; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyGrayButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2a6bbf; color: white; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Grayscale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyInvertButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1f4e91; color: white; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyBlurButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #004080; color: white; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyRotateButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #00264d; color: white; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Rotate 90Â°</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4a90e2; color: white; border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPexing">
    <property name="title">
     <string>Pexing</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInvert">
    <property name="title">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBlur">
    <property name="title">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGray">
    <property name="title">
     <string>Gray</string>
    </property>
   </widget>
   <addaction name="menuPexing"/>
   <addaction name="menuInvert"/>
   <addaction name="menuBlur"/>
   <addaction name="menuGray"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
